01week/datatypes.js

```diff
@@ -0,0 +1,68 @@
+
+// #1 use the Date() method which returns day and time 
+
+function theDate() {
+  return Date();
+}
+

```

Comment:
Remember to call your function! Nothing will happen with your code unless it is called.